python/pyspark/sql/dataframe.py

```diff
@@ -861,11 +861,11 @@ def groupBy(self, *cols):
             Each element should be a column name (string) or an expression (:class:`Column`).
 
         >>> df.groupBy().avg().collect()
-        [Row(avg(age)=3.5)]
+        [Row(avg_c1=3.5)]
```

Comment:
I don't think `avg_c1` is better than `avg(age)`. I agree the sql string of typed aggregate is too verbose, but I prefer to only simplify it, but not all aggregegate expressions.